AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/Structure.dfy

```diff
@@ -411,6 +484,50 @@ module {:options "/functionSyntax:4" } Structure {
     ]
   }
 
+  function ReplaceMutableContext(
+    input: map<string, string>,
+    kmsArn: string,
+    customEncryptionContext: map<string, string>
+  ) : (output: map<string, string>)
```

Comment:
Will refactor to clarify what the input is.
The requires clause on line 493 makes it pretty clear it is `BranchKeyContext`,
but I can make that clearer.